ts/components/conversation/media-gallery/LoadingIndicator.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+export const LoadingIndicator = () => {
```

Comment:
Was this tested? I don't see any other mentions of `loading-widget` in the changes, and some CSS would need to be changed to make this show as expected.

Reply:
Nope, haven’t wired it up yet.